### PR TITLE
Fixes #226

### LINKED ISSUE #226: Broken link under 2020s2r1 in the wiki
**Is your feature request related to a problem? Please describe.**
A link is broken under the description of issues completed for 2 sprint 2020:
Go to: https://aau-giraf.github.io/wiki/releases/2020s2r1/
Click on API_client#78 (second and third last in the list)
Expect a 404 page not found

**Describe the solution you'd like**
Delete the links if the issue do not exist otherwise change to the correct link for the issue.

**Additional context**
This issue was found when reviewing issue 204 for pre-release sprint 3 2020
https://github.com/aau-giraf/wiki/issues/204

### PR DESCRIPTION
**Pull request by Nannanas**
Update 2020s2r1

fix #226

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw610f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw611f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

File: docs/releases/2020s2r1.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ These are the user stories that where included in 2020s2r1:
 | [Web-API#78](https://github.com/aau-giraf/web-api/issues/78)                                | Group sw607f20         |
 | [Web-API#94](https://github.com/aau-giraf/web-api/issues/94)                                | Group sw616f20         |
 | [Web-API#96](https://github.com/aau-giraf/web-api/issues/96)                                | Group sw616f20         |
-| [API_client#78](https://github.com/aau-giraf/api_client/issues/78)                          | Group sw607f20         |
-| [API_client#78](https://github.com/aau-giraf/api_client/issues/78)                          | Group sw610f20         |
+| [API_client#24](https://github.com/aau-giraf/api_client/issues/24)                          | Group sw607f20         |
+| [API_client#33](https://github.com/aau-giraf/api_client/issues/33)                          | Group sw610f20         |
```
Comment (antonhpalmer) [inline docs/releases/2020s2r1.md]: From what I can tell, @Laexander created this issue and he is in group 15
```suggestion
| [API_client#33](https://github.com/aau-giraf/api_client/issues/33)                          | Group sw615f20         |
```

File: docs/releases/2020s2r1.md [hunk 2]
```diff
@@ -39,8 +39,8 @@ These are the user stories that where included in 2020s2r1:
 | [Web-API#78](https://github.com/aau-giraf/web-api/issues/78)                                | Group sw607f20         |
 | [Web-API#94](https://github.com/aau-giraf/web-api/issues/94)                                | Group sw616f20         |
 | [Web-API#96](https://github.com/aau-giraf/web-api/issues/96)                                | Group sw616f20         |
-| [API_client#78](https://github.com/aau-giraf/api_client/issues/78)                          | Group sw607f20         |
-| [API_client#78](https://github.com/aau-giraf/api_client/issues/78)                          | Group sw610f20         |
+| [API_client#24](https://github.com/aau-giraf/api_client/issues/24)                          | Group sw607f20         |
```
Comment (gitkshn) [inline docs/releases/2020s2r1.md]: Vi (gruppe 10) har lavet issue 24
```suggestion
| [API_client#24](https://github.com/aau-giraf/api_client/issues/24)                          | Group sw610f20         |
```

Comment (gitkshn) [reply docs/releases/2020s2r1.md]: Jeg går ud fra at "Developed by" betyder dem der har fixet issuet

File: docs/releases/2020s2r1.md (hunk 1 above)
Comment (Nannanas) [reply docs/releases/2020s2r1.md]: Yes group 15 made the issue, but group 7 fixed it. It should be correct now:
https://github.com/aau-giraf/api_client/issues/33
https://github.com/aau-giraf/api_client/issues/24

**Review by gitkshn — APPROVED** [review]
Har kørt det lokalt, og linksne virker

**Review by antonhpalmer — APPROVED** [review]

**Pull request merged**